themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties

```diff
@@ -246,4 +267,140 @@ havePermissionRequestsWaitingForApproval=Masz {0} żądań uprawnień oczekując
 clickHereForDetails=Więcej szczegółów...
 resourceIsNotBeingShared=Zasób nie jest współdzielony
 
-locale_fa=فارسی
\ No newline at end of file
+locale_ar=عربي
+locale_ca=Català
+locale_cs=Čeština
+locale_de=Deutsch
+locale_en=English
+locale_es=Español
+locale_fr=Français
+locale_hu=Magyar
+locale_fa=فارسی
+locale_it=Italiano
+locale_ja=日本語
+locale_lt=Lietuvių
+locale_nl=Nederlands
+locale_no=Norsk
+locale_pl=Polski
+locale_pt-BR=Português (Brasil)
+locale_ru=Русский
+locale_sk=Slovenčina
+locale_sv=Svenska
+locale_th=ไทย
+locale_tr=Türkçe
+locale_zh-CN=中文简体
+locale_fi=Suomi
+
+# Applications
+applicationName=Nazwa
+applicationType=Typ aplikacji
+applicationInUse=Tylko w użyciu
+clearAllFilter=Wyczyść filtry
+activeFilters=Aktywne filtry
+filterByName=Filtruj wg nazwy ...
+allApps=Wszystkie aplikacje
+internalApps=Wewnętrzne aplikacje
+thirdpartyApps=Zewnętrzne aplikacje
+appResults=Wyniki
+clientNotFoundMessage=Nie znaleziono klienta
+
+# Linked account
+authorizedProvider=Dostawca logowania
+authorizedProviderMessage=Dostawca logowania  powiązany z kontem
```

Comment:
There are two spaces between "logowania"  but only one is required "powiązany"

Reply:
Corrected